python/pyspark/sql/functions.py

```diff
@@ -2070,6 +2070,8 @@ class PandasUDFType(object):
 
     GROUP_MAP = PythonEvalType.SQL_PANDAS_GROUP_MAP_UDF
 
+    GROUP_AGG = PythonEvalType.SQL_PANDAS_GROUP_AGG_UDF
```

Comment:
So I'm worried that it isn't clear to the user that this will result in a full-shuffle with no-partial aggregation. Is there maybe a place we can document this warning?

Reply:
Added in docstring of `pandas_udf` and `groupby().agg()`